rafs/src/builder/core/blob.rs

```diff
@@ -62,12 +62,14 @@ impl Blob {
                     if let Some(zran) = &ctx.blob_zran_generator {
                         let reader = zran.lock().unwrap().reader();
                         blob_ctx.compressed_blob_size = reader.get_data_size();
+                        blob_ctx.current_compressed_offset = blob_ctx.compressed_blob_size;
                         if blob_ctx.blob_id.is_empty() {
                             let hash = reader.get_data_digest();
                             blob_ctx.blob_id = format!("{:x}", hash.finalize());
                         }
                     } else if let Some(tar_reader) = &ctx.blob_tar_reader {
                         blob_ctx.compressed_blob_size = tar_reader.position();
+                        blob_ctx.current_compressed_offset = blob_ctx.compressed_blob_size;
```

Comment:
This needs to be backported. Could you send it via a seperate PR? /cc @imeoer

Reply:
I was going to split it out as a dedicated PR, but found it's a false alarm:(
The assertion should be
```
assert!(self.current_compressed_offset >= self.compressed_blob_size);
```
instead of
```
assert!(self.current_compressed_offset > self.compressed_blob_size);
```

So the fixup is unnecessary:)